core/src/main/java/org/neo4j/ogm/context/GraphEntityMapper.java

```diff
@@ -160,6 +155,53 @@ public GraphEntityMapper(MetaData metaData, MappingContext mappingContext) {
         return results;
     }
 
+    /**
+     * Executes method annotated with @PostLoad on node and relationship entities with given ids
+     *
+     * This method must be executed after processing all result rows:
+     * - post load must be called after the entity is fully hydrated
+     * - to avoid executing post load multiple times on same entity
+     *
+     * @param nodeIds nodeIds
+     * @param edgeIds edgeIds
+     */
+    public void executePostLoad(Set<Long> nodeIds, Set<Long> edgeIds) {
+        Set<Long> seenIds = new HashSet<>();
```

Comment:
Maybe because it's late but I don't see the purpose of `seenIds` as method parameters are Sets. Some old code remaining ?

Reply:
Good catch!! It also made me discover another but with @PostLoad method, will push fix in another commit.